tests/ethpm/utils/test_manifest_validation_utils.py

```diff
@@ -7,10 +7,66 @@
     validate_manifest_against_schema,
     validate_manifest_deployments,
     validate_manifest_exists,
+    validate_raw_manifest_format,
 )
 from ethpm.validation import validate_manifest_version
 
 
+@pytest.fixture
+def all_raw_strict_manifests(package_names):
+    manifests = []
```

Comment:
Just my general dislike for mutation.  Converting this to use `to_tuple` and using a `yield` pattern should work here.